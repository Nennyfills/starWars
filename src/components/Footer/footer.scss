.footer {
  background-color: #834b73;
  padding: 0.1rem;
  box-shadow: hue;
  z-index: 10;
  text-align: center;
  position: fixed;
  left: 0;
  bottom: 0;
  height: 3rem;
  width: 100%;
  box-shadow: 0 0 3px;
  font-size: 0.8rem;
  font-weight: 500;
  color: #ffffff;
}
