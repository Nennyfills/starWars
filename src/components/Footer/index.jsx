import React from 'react';

import './footer.scss';

/**
 * Footer Component
 *  @param {string} message
 *  @returns footer element
 */
const Footer = () => {
  const year = new Date().getFullYear();

  return (
    <div className="footer">
      <h4 className="footer_fonts">
        Copyright &copy; Star Wars
        {year}
      </h4>
    </div>
  );
};

export default Footer;
