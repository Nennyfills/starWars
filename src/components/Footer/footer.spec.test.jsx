import React from 'react';

import Footer from './index';


describe('Footer Component', () => {
  it('should mount the Footer wrapper', () => {
    const wrapper = mount(<Footer />);

    expect(wrapper.find('.footer')).toBeDefined();
    expect(wrapper).toMatchSnapshot();

    // expect(wrapper.find('.vertical_design')).toBeDefined();
  });
});
