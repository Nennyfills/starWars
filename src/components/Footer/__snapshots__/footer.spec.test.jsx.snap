// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component should mount the Footer wrapper 1`] = `
<Footer>
  <div
    className="footer"
  >
    <h4
      className="footer_fonts"
    >
      Copyright Â© Star Wars
      2020
    </h4>
  </div>
</Footer>
`;
